lib/compiler.js

```diff
@@ -21,13 +21,17 @@ exports.compile = function (code, options) {
 
   const result = { ast: null, code };
 
-  if (! getOption(options, "force") &&  ! importExportRegExp.test(code)) {
+  if (! getOption(options, "force") &&
+      ! importExportRegExp.test(code)) {
+    // Let the caller know the result is no different from the input.
+    options.identical = true;
```

Comment:
Maybe a different kind of result like an object with a `response` or smth that can also have other data props associated with it.